DOI.js

```diff
@@ -154,6 +154,10 @@ function doWeb(doc, url) {
 	var dois = getDOIs(doc);
 	Z.debug(dois);
 	var providers = [
+		{
+			id : "b28d0d42-8549-4c6d-83fc-8382874a5cb9",
+			name : "DOI.org"
```

Comment:
* Remove Crossref + Datacite options
* possibly adjust name or add comment about the name of the translator